Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm

```diff
@@ -84,7 +84,7 @@ static inline void updateQueryForGroupIfNecessary(NSMutableDictionary *dictionar
 }
 #endif
 
-static RetainPtr<NSData> produceClientDataJson(_WKWebAuthenticationType type, NSData *challenge, NSString *origin)
+static RetainPtr<NSData> produceClientDataJson(_WKWebAuthenticationType type, NSData *challenge, NSString *origin, NSString *topOrigin = nil, WebAuthn::Scope crossOrigin = WebAuthn::Scope::SameOrigin)
```

Comment:
Maybe we can rename `crossOrigin` to `scope` for clarity.

Reply:
This is what the AuthExp team asked for

Reply:
@abigailfox This is internal code and is is a scope, it should be named scope. This had nothing to do with the AuthExp team or the API.